utils/auth.js

```diff
@@ -16,10 +16,9 @@
 
 'use strict';
 
-const GoogleAuth = require('google-auth-library');
+const {OAuth2Client} = require('google-auth-library');
```

Comment:
Hi, we just updated the linter to enforce spacing for destructuring objects. It was to enforce consistency for module imports like this as when there are multiple destructured properties, it is more readable with open spacing.

Can you change it to:
```javascript
const { OAuth2Client } = require('google-auth-library');
```

Reply:
Done.